modGui/src/org/aion/gui/model/ConsoleTail.java

```diff
@@ -0,0 +1,74 @@
+package org.aion.gui.model;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class ConsoleTail {
+    private final ScheduledExecutorService timer;
```

Comment:
unused

Reply:
Oops, in constructor I accidentally use the reference from the constructor arguments, instead of the member variable.  Will fix that.